src/plugins/vis_type_vega/public/vega_view/vega_map_view.js

```diff
@@ -102,6 +102,7 @@ export class VegaMapView extends VegaBaseView {
     // let maxBounds = null;
     // if (mapConfig.maxBounds) {
     //   const b = mapConfig.maxBounds;
+    // eslint-disable-next-line no-undef
```

Comment:
Is this necessary? Seems like this commented out code can be removed anyway.

Reply:
It was on the chance someone ever uncommented it. I don't know the history of why it was commented out and left in. In general I think retaining commented code is unnecessary (that's what git is for, seeing code history) but I didn't touch it otherwise since it wasn't the focus of this PR